Documentation/install/upgrade.rst

```diff
@@ -112,14 +143,24 @@ Kubernetes resources are updated accordingly to version you are upgrading to:
 
 .. include:: ../gettingstarted/k8s-install-download-release.rst
 
-Generate the required YAML file and deploy it:
+Deploy Cilium release via Helm:
```

Comment:
Note: The section header above here still refers to "Generate YAML Using Helm". Depending on what conclusion you come to on upgrade, you may want to consider updating it to something a little bit more clear.